config/x86/kernel-config

```diff
@@ -1,8 +1,8 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86 5.10.1 Kernel Configuration
+# Linux/x86 5.10.4 Kernel Configuration
 #
-CONFIG_CC_VERSION_TEXT="gcc (GCC) 8.3.0"
+CONFIG_CC_VERSION_TEXT="gcc (Debian 8.3.0-6) 8.3.0"
```

Comment:
It would be probably better to keep old value on this

Reply:
No problem with that. I just did a quick menuconfig and didn't edit the .config by hand.